techniques/html/H60.html

```diff
@@ -21,14 +21,14 @@ <h3>The WCAG 2 Glossary.</h3>
       <section class="procedure"><h3>Procedure</h3>
          <p>For any set of words and their definitions that are meant to serve as a glossary: </p>
          <ol>
-            <li>Check that the <code class="el">head</code> section of the web page that contains words,
-                  phrases or abbreviations defined in a glossary contains a <code class="el">link</code>
+            <li>Check that the <code>head</code> section of the web page that contains words,
+                  phrases or abbreviations defined in a glossary contains a <code>link</code>
                   element</li>
-            <li>Check that the <code class="el">link</code> element has attribute
+            <li>Check that the <code>link</code> element has attribute
                   
-										<code class="att">rel</code>="glossary"
+										<code>rel</code>="glossary"
```

Comment:
```suggestion
										<code>rel="glossary"</code>
```

and could benefit from highlighting